books/free-programming-books-subjects.md

```diff
@@ -314,9 +314,12 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Learning Modern 3D Graphics Programming](https://web.archive.org/web/20150225192611/http://www.arcsynthesis.org/gltut/index.html) - Jason L. McKesson (draft)
 * [Notes for a Computer Graphics Programming Course](https://www.cs.csustan.edu/~rsc/CS3600F00/Notes.pdf) - Dr. Steve Cunningham (PDF)
 * [OpenGL](http://www.songho.ca/opengl/index.html) - Concepts and illustrations
+* [Physically Based Rendering, Third Edition: from Theory to Implementation](https://www.pbr-book.org/3ed-2018/contents) - Matt Pharr
```

Comment:
Another good addition. I'd use https://www.pbr-book.org as the link, and pay attention to the authorlist!